src/components/SwipeableView/index.ios.js

```diff
@@ -0,0 +1,40 @@
+import React, {PureComponent} from 'react';
+import {PanResponder, View} from 'react-native';
+import PropTypes from 'prop-types';
+
+const propTypes = {
+    children: PropTypes.element.isRequired,
+
+    // callback whenever swipe down is registered
+    onSwipeDown: PropTypes.func.isRequired,
+};
+
+class SwipeableView extends PureComponent {
+    constructor(props) {
+        super(props);
+        this.panResponder = PanResponder.create({
+
+            // The PanResponder gets focus only when the y-axis movement is over 3 pixels
```

Comment:
Why 3px ? Maybe we can make this more obvious by using a constant that describes what the 3 represents e.g. `minimumPixelDistance`.

Reply:
I found 10 to be a bit too much. It was based on hit and trial. Yup, will make amends accordingly to have it as `minimumPixelDistance`